src/style.css

```diff
@@ -0,0 +1,40 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.counteiner {
+  display: inline-block;
+}
+
+.stars {
+  height: 16px;
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background: url(images/star.svg) no-repeat center;
+  margin-right: 4px;
```

Comment:
last star shouldn't has margin-right

Reply:
without without margin-right the test does not pass

![margin-right](https://user-images.githubusercontent.com/119884765/215243990-6cb097ef-f8f0-42d4-b3f5-6cf137ddb469.jpg)
![margin-right2](https://user-images.githubusercontent.com/119884765/215244002-edbe9670-fabb-41e7-aa02-acf59a886437.jpg)

Reply:
Not fixed.
Remove div with class counteiner and set last star margin-right: 0.
If you will do both fixes tests should pass